app/models/contestant.rb

```diff
@@ -1,3 +1,9 @@
 class Contestant < ApplicationRecord
+  has_many :contestant_projects
+  has_many :projects, through: :contestant_projects
   validates_presence_of :name, :age, :hometown, :years_of_experience
+
+  def projects_names
+    self.projects.pluck(:name).to_sentence(two_words_connector: ', ')
```

Comment:
As we discussed, this is totally okay to use in a model method! I am not a fan of separating the `.to_sentence` out and just using that in the view because it makes testing more cumbersome. Tacking it on the end of the query is still MVC-compliant IMO.